locale/afrux-theme-base.yml

```diff
@@ -0,0 +1,33 @@
+afrux-theme-base:
+  admin:
+    extensions:
+      extensions: Erweiterungen
+      title: Erweiterungen
+      desc: Liste aller installierten Erweiterungen.
+    extension:
+      sections:
+        content: Basis Einstellungen
+        permissions: Berechtigungen
+
+    header:
+      go_back: Zurück zum Forum
+
+    users:
+      grid:
+        columns:
+          avatar:
+            title: Avatar
+
+    settings:
+      edit_footer_links: Bearbeite Footer
+      footer_description: Footer Beschreibung
+      footer_bottom_line: Footer Endeffekt
+      hero_banner: Willkommen Hero Bild
```

Comment:
I guess we must keep this, I'm not sure how to translate "Hero" here. I don't even know why Flarum uses that word in the first place.